Updates to Windows startup issues (#249)
ankitects/anki-manual: src/platform/windows/startup-issues.md

## No error, but app does not appear

If you start Anki and it fails to appear, without

> any error message, you can try:

Reviewer: ```suggestion
any error message, you can try the following:
```

better to complete the sentence maybe, and it helps keep the gerunds out, which is confusing for some ESLs and we also normally don't use them in Manual.

> - Disconnecting multiple/external displays.

Reviewer: ```suggestion
- Disconnect multiple/external displays.
```

> - Installing the [latest Anki version](https://apps.ankiweb.net/) (try both qt6 and qt5).

Reviewer: ```suggestion
- Install the [latest Anki version](https://apps.ankiweb.net/) (try both qt6 and qt5).
```

> - Adjusting [your decimal separator](https://forums.ankiweb.net/t/windows-update-broke-anki/1822/75), if it is not a period.

Reviewer: ```suggestion
- Adjust [your decimal separator](https://forums.ankiweb.net/t/windows-update-broke-anki/1822/75) if it's is not a period.
```

Author: Accepted in part.

> - Installing the old [2.1.35-alternate build](https://github.com/ankitects/anki/releases/tag/2.1.35) of Anki.

Reviewer: ```suggestion
- Install the old [2.1.35-alternate build](https://github.com/ankitects/anki/releases/tag/2.1.35) of Anki.
```

`C:\Program Files\Anki` and `C:\Program Files (x86)\Anki`), they may be left in a
non-working state, and Anki may refuse to start without showing an error message.

> Try uninstalling all copies - from Windows Settings > Apps & features (or Apps > Installed apps), or by running `uninstall.exe` in each Anki program

Reviewer: Generally we wrap UI elements in quotes (unlike most modern styles where bold is used). Is this better?

```suggestion
Try uninstalling all copies of Anki from your computer. To do this, Go to "Settings" > "Apps & features" (or "Apps" > "Installed apps"), or run `uninstall.exe` in each Anki program
```

Author: Oof, that's too many quotation marks -- I can't do it! 😅 Is that the house style? I haven't noticed such an abundance of quotes. 

Updated in part

Reviewer: I can understand. The house style here isn't consistent. I would've expected UI elements to be quoted in most instances but many a times we don't (arbitrarily). And in this situation it certainly will look cluttered. Google/MS/IBM etc. bold UI element names in their docs. I'm fine either way though, just should be consistent.

folder. Afterward, install Anki again.

## Debugging

Starting Anki from a terminal may reveal a bit more information about some
errors. After installing the latest Anki version and ensuring all Windows

> updates are installed, instead of running Anki directly, press the <kbd>Windows</kbd> key (or open the Start menu), type `cmd`, and launch Command Prompt. When the terminal window opens, paste (or type) the following command, and press <kbd>Enter</kbd>. (The path will be different if Anki is installed in a location that is not the default.)

Reviewer: ```suggestion
updates are installed, instead of running Anki directly, press the <kbd>Windows</kbd> key (or open the "Start" menu), type `cmd`, and launch Command Prompt. When the terminal window opens, input the following command, and press <kbd>Enter</kbd>. (The path will be different if Anki is installed in a location that is not the default.)
```

Author: "Input" makes it sound too technical. I'm trying to beginner-proof this description.

Reviewer: If you wish you can say enter info. Paste is decent too. Type is a bit non-inclusive.

Author: I'll just go with "paste" -- the block has a copy button, and we'll avoid more errors that way.
